src/java_bytecode/character_refine_preprocess.cpp

```diff
@@ -132,9 +132,13 @@ codet character_refine_preprocesst::convert_digit_char(
   conversion_inputt &target)
 {
   const code_function_callt &function_call=target;
-  assert(function_call.arguments().size()==2);
+  PRECONDITION(function_call.arguments().size()>=1);
```

Comment:
Is the case it should be either 1 or 2 - you might consider tightening this condition to `>= 1 && <= 2`